README.md

```diff
@@ -63,3 +63,33 @@ In order to prepare an AWS instance, it needs to have OpenShift pre-installed an
 	cloud := cloudprepareaws.NewCloud(
 		gwDeployer, ec2.New(awsSession), infraID, region, gwInstanceType)
 ```
+
+### GCP
+
+In order to prepare a GCP instance, it needs to have OpenShift pre-installed and running.
```

Comment:
I hope in the future we can find community support to extend this for GKE, or other distros also.